Source/WebCore/html/ImageInputType.cpp

```diff
@@ -175,6 +175,8 @@ unsigned ImageInputType::height() const
     ASSERT(element());
     Ref<HTMLInputElement> element(*this->element());
 
+    ContentVisibilityForceLayoutScope scope(element.get());
```

Comment:
These seems to be sprinkled around fairly arbitrarily. How will future patches know when this is needed? We need to factor the code in such a way that it's hard to make mistakes in future.

Reply:
For this case I was not sure ContentVisibilityForceLayoutScope made sense in updateLayout, hence I kept it out, but looking at updateLayout again, it does make sense I think to use ContentVisibilityForceLayoutScope in updateLayout (and only there), so I was able to remove all other ContentVisibilityForceLayoutScope uses.